src/Swarm/TUI/Controller.hs

```diff
@@ -282,7 +282,7 @@ handleMainEvent ev = do
         WorldPanel -> do
           mouseCoordsM <- Brick.zoom gameState (mouseLocToWorldCoords mouseLoc)
           uiState . uiWorldCursor .= mouseCoordsM
-        REPLInput -> handleREPLEvent ev
+        REPLPanel -> handleREPLEvent ev
```

Comment:
So this still isn't a perfect solution: now, clicking anywhere on the entire REPL panel can cause the cursor to jump around inside the input.  To see this, type some long input, click on a different panel, and then click somewhere inside the REPL panel on the left side, but not on the line of input.  The cursor will jump to somewhere in the middle of the input --- and the position where it jumps doesn't even correspond exactly with the x-coordinate of where you click.

I was thinking of keeping `REPLInput` and simply wrapping the REPL editor widget in `clickable REPLInput` --- does that not work?